add Early Adopter list (#5888)
getsentry/sentry-docs: src/docs/product/accounts/early-adopter.mdx

---
title: "Early Adopter Features"
sidebar_order: 60
description: "Learn which features are currently in the early adopter phase."
---

This page lists the features that you'll have access to when you enable the "Early Adopter" toggle in **Settings > General Settings**:

![The Early Adopter toggle enabled in settings.](early-adopter-toggle.png)

Limitations:

> - This list does not include alphas, closed betas, or limited availbility features that require manual opt-in and/or are not controlled by the "Early Adopter" setting.

Author: ```suggestion
- This list does not include new features that aren't controlled by the "Early Adopter" setting, such as alphas, closed betas, or limited availability features that require manual opt-in.
```

- This list is not guaranteed to be 100% up-to-date, but it is monitored and updated frequently.

## Current Early Adopter Features

- [Grouping Breakdown](/product/data-management-settings/event-grouping/grouping-breakdown/)
- [Issue Reprocessing](/product/issues/reprocessing/)

> - [Span Summary](/product/performance/transaction-summary/#span-summary)

Author: ```suggestion
- [Span Summary](/product/performance/transaction-summary/#span-summary)
- Updated Org URLs - Org URLs are being updated to a new format. 
```

---
title: "Early Adopter Features"
sidebar_order: 60
description: "Learn which features are currently in the early adopter phase."
---

This page lists the features that you'll have access to when you enable the "Early Adopter" toggle in **Settings > General Settings**:

![The Early Adopter toggle enabled in settings.](early-adopter-toggle.png)

Note that features are sometimes released to early adopters in waves, so you may not see a feature immediately upon enabling the "Early Adopter" setting.

Limitations:

- This list does not include new features that aren't controlled by the "Early Adopter" setting, such as alphas, closed betas, or limited availability features that require manual opt-in.
- This list is not guaranteed to be 100% up-to-date, but it is monitored and updated frequently.

## Current Early Adopter Features

- [Grouping Breakdown](/product/data-management-settings/event-grouping/grouping-breakdown/)
- [Issue Reprocessing](/product/issues/reprocessing/)
- [Span Summary](/product/performance/transaction-summary/#span-summary)

> - Org URLs - Org URLs are being updated to a new format. 

Reviewer: ```suggestion
- Organization Domains - Instead of being a path segment, organization slugs are used as a subdomain for the Sentry interface. Eg. `https://acme.sentry.io/issues/` instead of `https://sentry.io/organizations/acme/issues/`. API URLs are unchanged. 
```

> - Organization Domains - Instead of being a path segment, organization slugs are used as a subdomain for [sentry.io](https://sentry.io). For example, `https://acme.sentry.io/issues/` will replace `https://sentry.io/organizations/acme/issues/`. API URLs are unchanged. 

Author: ```suggestion
- Organization Domains - Instead of being a path segment, organization slugs are used as a subdomain for [sentry.io](https://sentry.io). For example, `https://acme.sentry.io/issues/` replaces `https://sentry.io/organizations/acme/issues/`. API URLs are unchanged. 
```
